benchmarking/driver/benchmark_driver.py

```diff
@@ -297,6 +297,10 @@ def _retrieveMeta(info, benchmark, platform, framework, backend):
         meta["identifier"] = test["identifier"]
     else:
         meta["identifier"] = meta["net_name"]
+    if 'type' in test['inputs']['data']:
```

Comment:
data is the name of the blob. It can be anything in the model. So you cannot just check data here. 

Different inputs may have different input types. What is the purpose of this change?